students/charlie_england/lesson01/assignment/inventory_management/electric_appliances_class.py

```diff
@@ -0,0 +1,29 @@
+"""Electric appliances class"""
+from inventory_class import Inventory
```

Comment:
Since the files are mainly going to be running in the assignment directory, it is a good idea to modify your import to not be relative:
from inventory_management.inventory_class import Inventory